### ISSUE TITLE
Template sensor template picture not changing

### ISSUE THREAD

**Issue by klejejs** [estevez-dev/ha_client#369]
If template sensor picture changes, app doesn't show it until you manually press refresh button.

**Comment by estevez-dev** [comment]
The problem here that if there was no state change for sensor, the app will not be notified about changes. Does the picture changes with state?

**Comment by klejejs** [comment]
Yes it does. I have set up a template sensor. Here is the code:

```
krisjanis_device_tracker:
  friendly_name: Krisjanis device tracker sensor
  value_template: "{{ states('device_tracker.krisjanis') }}"
  entity_picture_template: >
    {% if is_state('device_tracker.krisjanis', 'home') %}
      /local/images/profile/krisjanis1.jpg
    {% else %}
      /local/images/profile/not_home.jpg
    {% endif %}
```

Another problem I encounter with this sensor is that in the app it doesn't show state history but in the web UI it does. Maybe this is related? Should I report this as a different issue?

**Comment by estevez-dev** [comment]
> I have set up a template sensor. Here is the code
Thanks, will be helpful

> Another problem I encounter with this sensor is that in the app it doesn't show state history but in the web UI it does. Maybe this is related? Should I report this as a different issue?
We already have it: #366

**Comment by klejejs** [comment]
Oh, I had forgotten that I already reported it😅

**Comment by estevez-dev** [comment]
Will be included in 0.8.0
